nomad/node_endpoint_test.go

```diff
@@ -1384,6 +1384,60 @@ func TestClientEndpoint_GetClientAllocs_Blocking(t *testing.T) {
 	}
 }
 
+func TestClientEndpoint_GetClientAllocs_WithMigrateTokens(t *testing.T) {
```

Comment:
Update this test to check the migration token is only created when necessary and for the previous nodes secret id